projects/plugins/boost/app/assets/src/js/stores/dismissed-alerts.ts

```diff
@@ -0,0 +1,52 @@
+import { SyncedStoreCallback } from '@automattic/jetpack-svelte-data-sync-client';
+import { z } from 'zod';
+import { jetpack_boost_ds } from './data-sync-client';
+
+const dismissedAlertsSchema = z.record( z.string().min( 1 ), z.boolean() );
+
+export const dismissedAlertsClient = jetpack_boost_ds.createAsyncStore(
+	'dismissed_alerts',
+	dismissedAlertsSchema
+);
+
+type MapType = z.infer< typeof dismissedAlertsSchema >;
+
+/**
+ * This function creates an SyncedStoreCallback.
+ * It's wrapped in a function so that pendingValues can be scoped to this function.
+ */
+function createDismissedAlertSyncAction() {
```

Comment:
I'm still learning my way around DS. Could you explain why this is necessary?

As I understand, we only have the one alert for the History graph. Are you future proofing this by making it support multiple alerts (for example - avoid racing conditions)?

Reply:
Since we are handling multiple dismissable alerts in multiple places, this would make them all manageable in the same place. I intend to remove different ways of dismissing alerts and make them consistent with DS. This will require a followup PR as you suggested.